# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1489623052 14400
#      Wed Mar 15 20:10:52 2017 -0400
# Node ID b7eaa87ffbe3b8fc47a1b223c1bb6cfc74655a94
# Parent  feaf0b0e0b1f72225e94e0c3ef8e23c45ad1dcb3
test/module.mk: Fix think-o in previous change.

diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -67,17 +67,17 @@ test/.gdbinit: etc/gdbinit
 
 define run-octave-tests
   cd test && $(SHELL) ../run-octave $(RUN_OCTAVE_OPTIONS) $(1) --norc --silent --no-history $(abs_top_srcdir)/test/fntests.m $(abs_top_srcdir)/test
   if $(AM_V_P); then \
     echo ""; \
     if [ -f test/fntests.log ]; then \
       echo "Contents of test/fntests.log:"; \
       echo ""; \
-      $(AWK) -f $(srcdir)/build-aux/show-failures.awk test/fntests.log; \
+      $(AWK) -f $(srcdir)/test/show-failures.awk test/fntests.log; \
     else \
       echo "test/fntests.log is missing!"; \
     fi; \
   fi
 endef
 
 check-local: $(GENERATED_TEST_FILES) | $(OCTAVE_INTERPRETER_TARGETS) test/$(octave_dirstamp)
 	$(call run-octave-tests)
